components/eam/cime_config/config_component.xml

```diff
@@ -136,7 +136,7 @@
       <value compset="20TR(?:SOI)?_EAM.*CHEMUCI.*LINOZ.*MAM5"  >20TR_eam_chemUCI-Linoz-mam5</value>
       <value compset="ARM95_EAM"                        >scam_arm95</value>
       <value compset="ARM97_EAM"                        >scm_arm97_chemUCI-Linoz-mam5-vbs</value>
-      <value compset="SCM_EAM"                          >scm_generic_chemUCI-Linoz-mam5-vbs</value>
+      <value compset="SCM_EAM"                          >scam_generic</value>
```

Comment:
@bogensch should we add a test of this compset?  We only test ARM97_EAM

Reply:
Yes, it's probably wise.  I'll try to get this done before I leave for vacation tomorrow afternoon... else expect a period of radio silence (FYI).

Reply:
Will need a new testmod for it, to bring in some of the settings specified in SCM case scripts to allow setup to proceed. (e.g., scmlat, scmlon)